diracx-client/src/diracx/client/patches/utils.py

```diff
@@ -322,3 +336,121 @@ def __init__(
     @property
     def client_id(self):
         return self._client_id
+
+
+class DeviceFlowErrorResponse(Model):
+    """TokenResponse.
+
+    All required parameters must be populated in order to send to Azure.
+
+    :ivar access_token: Access Token. Required.
+    :vartype access_token: str
+    :ivar expires_in: Expires In. Required.
+    :vartype expires_in: int
+    :ivar state: State. Required.
+    :vartype state: str
+    """
+
+    _validation = {
+        "error": {"required": True},
+    }
+
+    _attribute_map = {
+        "error": {"key": "error", "type": "str"},
+    }
+
+    def __init__(self, *, error: str, **kwargs: Any) -> None:
+        """
+        :keyword error: Access Token. Required.
+        :paramtype error: str
+        """
+        super().__init__(**kwargs)
+        self.error = error
+
+
+def build_token_request(**kwargs: Any) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+
+    accept = _headers.pop("Accept", "application/json")
+
+    # Construct URL
+    _url = "/api/auth/token"
+
+    _headers["Accept"] = _SERIALIZER.header("accept", accept, "str")
+
+    return HttpRequest(method="POST", url=_url, headers=_headers, **kwargs)
+
+
+class AuthOperations(AuthOperationsGenerated):
+    @distributed_trace
+    async def get_oidc_token(
+        self, vo: str, device_code: str, client_id: str, **kwargs
```

Comment:
`vo` does not seem to be used at all here.

```suggestion
        self, device_code: str, client_id: str, **kwargs
```